model/account.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+type Account struct {
+	ID       uint   `gorm:"primary_key"`
+	User     string `gorm:"not null;unique"`
+	Password string `gorm:"not null"`
+	Sex      string `gorm:"type:char(1)"`
+	Email    string `gorm:"unique"`
```

Comment:
I agree with you, but then, how we would handle this? I mean, if the owner opts in, how should we add the uniqueness constraint to the database?